<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTransformations</class>
 <widget class="QDialog" name="DialogTransformations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="labelAngleY">
         <property name="text">
          <string>Angle Y:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveScaleY" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAngleX">
         <property name="text">
          <string>Angle X:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveScaleX" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelScaleZ">
         <property name="text">
          <string>Scale Z:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelScaleY">
         <property name="text">
          <string>Scale Y:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelScaleX">
         <property name="text">
          <string>Scale X:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveAngleX" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="8">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>238</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveAngleZ" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveCenterX" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCenterZ">
         <property name="text">
          <string>Center Z:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveCenterY" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveScaleZ" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelCenterX">
         <property name="text">
          <string>Center X:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelAngleZ">
         <property name="text">
          <string>Angle Z:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="PanelPrimitiveValue" name="PanelPrimitiveAngleY" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>°</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>°</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>°</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTransformations</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTransformations</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
